README.md

```diff
@@ -45,6 +45,15 @@ You'll get better and better with experience.
   * basic concepts of modern CPU and computer architecture
   * basic concepts of disassembling and debugging of C code
   * _need help for translation_
+* [ARM Architecture Reference Manual, 2nd Edition](http://www.mypearsonstore.ca/bookstore/arm-architecture-reference-manual-9780201737196)
+  * Complete reference on ARM programming
+* [Computer Organization and Design: MIPS Edition](https://www.amazon.ca/Computer-Organization-Design-MIPS-Interface/dp/0124077269/)
```

Comment:
Please change form:
* Computer Organization and Design
  * [MIPS Edition](https://www.amazon.ca/Computer-Organization-Design-MIPS-Interface/dp/0124077269/)
  * [ARM Edition](https://www.amazon.ca/Computer-Organization-Design-ARM-Interface/dp/0128017333/)